server/migrations/20210922101959-create-categories.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.createTable('categories', {
+      id: {
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      catname: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      agencyId: {
+        allowNull: true, //to change to false once we implement this across all agencies
```

Comment:
```suggestion
        allowNull: false
```
I think this should be false from the get-go, you can't really have categories without an agency. An agency however can get away without having categories, particularly since posts already have a nullable categoryId.